packages/next/client/page-loader.js

```diff
@@ -227,6 +255,15 @@ export default class PageLoader {
     }
   }
 
+  createPreloadLink (url) {
```

Comment:
Let's move this helper top-level too since class syntax can be heavy. Will save us some bytes for IE.